docs/content/build.md

```diff
@@ -1,46 +1,54 @@
 **Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
 
------
-## Building your workers
+---
 
-Before running a [deployment of your game](deploy.md), you need to:
-1. Build the bridge and launch configuration of your [workers](workers.md)
-2. Prepare the build configuration of your workers
-3. Build your workers
+## How to build your game
 
-To build your workers, you don't need to know what the bridge and launch configurations are, but if you do want to know more about them, see the SpatialOS documentation on [bridge configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#bridge-configuration) and [launch configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-launch-configuration).
+Before running a [deployment of your game](deploy.md) locally or in the cloud, you need to:
+
+1. Build your workers.
+2. Build the bridge configuration and launch configuration of your [workers](workers.md).
+3. Prepare the build configuration of your workers.
```

Comment:
<3 the match between this and the titles.

Reply:
thanks, I did it myself

Reply:
You are brilliant.